korman/exporter/mesh.py

```diff
@@ -224,6 +224,10 @@ def _create_geospan(self, bo, mesh, bm, hsgmatKey):
         if mods.lighting.rt_lights:
             geospan.props |= plGeometrySpan.kPropRunTimeLight
 
+        # Attach a FogEnvironment if it's non-default
+        if mods.fogenv.fog_env and mods.fogenv.fog_env != bpy.context.scene.world:
```

Comment:
Considering all the access of `mods.fogenv`, it would be nice to cache that value for the (small) performance benefit and less verbosity.